joomla/joomla-cms
# Sample data: Escape quota in json data

Continuation of sample data files fixes started at PR #11334.
### Summary of Changes

In sql sample data files escape quota for json data.
### Testing Instructions

Code review or
1. Create clone of git joomla or download zip
2. Install  the latest joomla version with sample data "Learn Joomla English (GB) Sample Data"
   or "Test English (GB) Sample Data"
3. Login to backend and go to:
   /administrator/index.php?option=com_menus&view=item&layout=edit&id=252
4. You should see error.
5. Start new install  do the same but before choose sample data replace sample data sql file with patched equivalent. Then complete installation.
6. Login to backend and go to: /administrator/index.php?option=com_menus&view=item&layout=edit&id=252
   Now everything works as expected.
### Documentation Changes Required

No
